src/huggingface_hub/constants.py

```diff
@@ -21,9 +21,13 @@
 )
 
 REPO_TYPE_DATASET = "dataset"
-REPO_TYPES = [None, REPO_TYPE_DATASET]
+REPO_TYPE_SPACE = "space"
+REPO_TYPES = [None, REPO_TYPE_DATASET, REPO_TYPE_SPACE]
```

Comment:
(nit) This would be cleaner with a `REPO_TYPE_MODEL` eventually, no? (can be done later)